package-lock.json

```diff
@@ -47,7 +47,7 @@
                 "@elementor/externalize-wordpress-assets-webpack-plugin": "0.1.0",
                 "@elementor/extract-i18n-wordpress-experssions-webpack-plugin": "0.1.0",
                 "@elementor/generate-wordpress-asset-file-webpack-plugin": "0.1.0",
-                "@playwright/test": "^1.33.0",
+                "@playwright/test": "1.33.0",
```

Comment:
I don't think we're supposed to change `package-lock.json`

Reply:
it is updated automatically on any npm install when someone add/remove/update packages